src/mavsdk/core/include/mavsdk/mavsdk.h

```diff
@@ -153,8 +153,7 @@ class Mavsdk {
         GroundStation, /**< @brief SDK is used as a ground station. */
         CompanionComputer, /**< @brief SDK is used as a companion computer on board the MAV. */
         Camera, /** < @brief SDK is used as a camera. */
-        Custom /**< @brief the SDK is used in a custom configuration, no automatic ID will be
-                  provided */
+        RemoteId /**< @brief the SDK is used as a RemoteId system. */
```

Comment:
Hang on, what happens to `Custom`? We always need `Custom`.

Reply:
`Custom` just defaults to MAV_TYPE_GENERIC so it's misleading.